config/jobs/kubernetes/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -723,3 +723,80 @@ periodics:
     testgrid-tab-name: ci-cloud-provider-azure-multiple-zones
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: "Runs Kubernetes multiple availability zones tests with cloud-provider-azure (https://github.com/kubernetes/cloud-provider-azure) on VMSS."
+
+# scalability tests
+- interval: 3h
+  # ci-kubernetes-kubemark-100-gce runs kubemark tests with cloud-provider-azure of 100 nodes.
+  name: ci-kubernetes-kubemark-100-gce
+  tags:
+  - "perfDashPrefix: kubemark-100Nodes"
+  - "perfDashJobType: performance"
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+    preset-e2e-kubemark-common: "true"
+  annotations:
+    testgrid-dashboards: provider-azure-scalability
+    testgrid-tab-name: ci-cloud-provider-azure-scalability
+    testgrid-num-failures-to-alert: '1'
```

Comment:
Do we want to alert this often? Maybe 2 (or 3) is a saner threshold?

Reply:
> @nilo19 -- I've made a few comments and code suggestions.
> 
> I think it would be better to group the tests by Kubernetes version, instead of type.
> Then within each dashboard, have tabs for the various test types (like scalability).
> 
> Some of this is my best guess, based on other test configs I've seen, but Rita and Pengfei should definitely take a look as well.
> 
> Please also update the PR title, description, and commit message to be more descriptive, as this is a repo with multiple provider test configs.
> 
> Something like:
> 
> ```
> provider/azure: Add kubemark 100 test configs for kubernetes/cloud-provider-azure
> ```
> 
> /assign @ritazh @feiskyer

Thanks a lot @justaugustus for your suggestions.

Reply:
yea, let's change to 3